generators/entity/templates/src/test/java/package/web/rest/_EntityResourceIntTest.java

```diff
@@ -51,9 +50,8 @@
  * @see <%= entityClass %>Resource
  */
 @RunWith(SpringJUnit4ClassRunner.class)
-@SpringApplicationConfiguration(classes = <%= mainClass %>.class)
+@SpringBootTest(classes = <%= mainClass %>.class)
 @WebAppConfiguration
```

Comment:
@mraible
From what I read, the @WebAppConfiguration is not needed when using @SpringBootTest. Also you can replace 'SpringJUnit4ClassRunner.class' with 'SpringRunner.class'.